Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -579,6 +582,7 @@ def _uniqueFirstLayers(layerList):
 ]))
 
 tracksValidation = cms.Sequence(
+    highPtJets +
```

Comment:
Please add `highPtJets` (and likely later clones for different jet collections (https://github.com/cms-sw/cmssw/pull/25514#issuecomment-449040541) to `tracksPreValidation` task instead.

Reply:
done, how can I test the four paths?